documents/2-RTB/Documentazione_Interna/Norme_di_Progetto.typ

```diff
@@ -118,3 +129,182 @@ Il team si aspetta di rispettare in ogni momento gli standard di qualità defini
+11. Chiusura della Issue
+12. Spostamento della Issue da `In Progress` a `Done`
+13. Chiusura PR e merge su branch `Develop`
+14. Eventuale eliminazione del branch (solo se non deve essere riutilizzato)
+
+*Milestones* \
+Il gruppo, dopo una pianificazione accurata, ha deciso di impostare le milestones ad "artefatti" e non a "sprint".
+Il gruppo ritiene che le Milestones indichino qualcosa di completato, quindi è stato deciso di ritenere una milestone completata solamente quando un artefatto è stato completato e verificato. Al contrario, è stata impostata una data di inizio e una data di fine alla singola issue, di modo che sia comunque monitorabile nel tempo l'avanzamento del progetto.
+
+=== Coordinamento
+E' l'attività responsabile della gestione delle comunicazioni e incontri, sia interni che esterni. Questo compito assume un ruolo fondamentale nella corretta riuscita del progetto, garantendo l'efficienza e l'efficacia del Team stesso.
+
+Le attività di coordinamento comprendono le comunicazioni interne ed esterne, l'organizzazioni delle riunioni e la gestione delle scadenze.
+
+==== Comunicazioni
+
+===== Interne
+
+Le comunicazioni interne avvengono con due modalità: le comunicazioni scritte, formali e non, avvengono attraverso Telegram. Le comunicazioni verbali, quindi le riunioni, la condivisione di file, di codice, e di qualsiasi altra risorsa avvengono attraverso Discord.
+
+Su Telegram sono affrontate solamente questioni di rapida risoluzione, e comunicazioni off-topic. Le decisioni più strutturate, come elencato sopra, verranno prese su discord in presenza di tutti i memebri del Team durante la riunione settimanale.
+
+===== Esterne
+
+Tutte le comunicazioni esterne sono gestite dal Responsabile del Progetto, previa consultazione con il Team e visione della comunicazione stessa. Vengono effettuale esclusivamente tramite email attraverso questo indirizzo: #link("mailto:" + g.mail).
+
+==== Riunioni
+
+Il responsabile attuale ha il compito di introdurre i punti che saranno trattati nella riunione e di redigere il verbale interno, oltre che a decidere chi lo verificherà. La riunione avviene su Discord, in un canale dedicato.
+
+===== Interne
+
+Le riunioni interne avvengono settimanalmente, il giorno e l'ora sono decisi dal Responsabile e comunicati al Team in base alle esigenze di ogni membro. Queste riunioni servono per discutere lo stato del progetto, le attività svolte e da svolgere, e per prendere decisioni importanti.
+
+===== Esterne
+
+Le riunioni esterne avvengono con il Proponente e con i Committenti ad orari prestabiliti: i martedì dalle 17 alle 18. Le riunioni esterne sono effettuate su Google Meet. Queste riunioni sono gestite dal Responsabile del Progetto, che si occupa di organizzare l'incontro, di redigere l'ordine del giorno e di redigere il verbale esterno. Quest'ultimo viene condiviso con il Team e con il Proponente.
```

Comment:
297:
Le riunioni esterne avvengono con il Proponente e con i Committenti ad orari prestabiliti: i martedì dalle 17 alle 18
->
Le riunioni esterne che coinvolgono il Proponente e i Committenti si tengono generalmente ogni due settimane, il martedì dalle 17:00 alle 18:00 (con eventuali modifiche di data o orario in caso di necessità).

GitHub -> #glossary("Google Meet")
e di redigere il verbale esterno -> e il verbale esterno

Reply:
done